### PR TITLE
[fr] Translate new phrases

### PR DESCRIPTION
**Pull request by noweh**

### CODE REVIEW

**Review by andrey-helldar — APPROVED** [review]

File: locales/fr/fr.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
     "Bahrain": "Bahreïn",
     "Bangladesh": "Bangladesh",
     "Barbados": "Barbades",
-    "Before proceeding, please check your email for a verification link.": "Before proceeding, please check your email for a verification link.",
+    "Before proceeding, please check your email for a verification link.": "Avant de continuer, merci de vérifier votre e-mail contenant un lien de verification.",
```
Comment (caouecs) [inline locales/fr/fr.json:84]: verification => vérification

**Review by caouecs — CHANGES_REQUESTED** [review]
Juste une erreur d'accent.

Merci par avance

**Review by olsza — COMMENTED** [review]
hey @noweh, you don't need to add the files from docs/statuses/fr.md directory to PR... they are generated automatically when merge PR.

Ps. thanks for your help i.e. translation :)

**Review by olsza — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Merci

**Pull request merged**